docs/quick_start.md

```diff
@@ -23,15 +33,45 @@ touch Pipfile && pipenv --python 3.10 && pipenv shell
 pip install open-autonomy
 ```
 
+## Deploy a local agent service
+
+Follow the steps indicated in the diagram to define and build and deploy locally an example agent service.
+In this case, we consider the [Hello World agent service](https://docs.autonolas.network/service_example/), which is a service for demonstration purposes.
+
+The steps defined in the figure above are materialized as follows:
+
+1. **Component provision:** The developer ensures that the agent components are available, either by programming them from scratch using the {{open_aea}} framework, or fetching them from the Component Registry. Each agent service needs one or more underlying {{fsm_app}}(s) that define the service business logic. The {{open_autonomy}} framework contains packages with base classes that help define {{fsm_app}}s.
+    - In our case, we don't need to provision or define any component, since we will fetch an agent from the registry in the next step.
+2. **Agent definition:** The developer ensures that the agents that make up the service are available, either by defining them  using the {{open_aea}} framework, or by fetching them from the Agent Registry.
+    - Execute the following commands:
```

Comment:
For the service, we just need to fetch the service and build it from service config. That then builds the individual agents. So this is misleading.

Reply:
Indeed you are right. This was not a finalized branch, though. The CLI is under refactoring during this week, and this will be updated accordingly after this takes place.

Reply:
As agreed on a conversation, this PR will be fixed to reflect current status. Upcoming updates in the CLI will be reflected in new doc PRs